src/components/common/ColonyHome/ColonyHome.tsx

```diff
@@ -1,45 +1,32 @@
-import clsx from 'clsx';
 import React from 'react';
 import { defineMessages } from 'react-intl';
-import { useLocation } from 'react-router-dom';
 
 import ColonyActionsTable from '~common/ColonyActionsTable';
-import { ACTION } from '~constants/actions';
-import { useActionSidebarContext } from '~context';
 import { useSetPageBreadcrumbs } from '~context/PageHeadingContext/hooks';
 import { useColonyContext, useColonySubscription, useMobile } from '~hooks';
 import {
   useCreateTeamBreadcrumbs,
   useGetSelectedTeamFilter,
 } from '~hooks/useTeamsBreadcrumbs';
 import {
-  COLONY_MEMBERS_ROUTE,
-  COLONY_DETAILS_ROUTE,
   // @BETA: Disabled for now
   // COLONY_TEAMS_ROUTE,
-  COLONY_AGREEMENTS_ROUTE,
   COLONY_ACTIVITY_ROUTE,
-  COLONY_BALANCES_ROUTE,
   TEAM_SEARCH_PARAM,
 } from '~routes';
-import Numeral from '~shared/Numeral';
 import { formatText } from '~utils/intl';
-import { getTokenDecimalsWithFallback } from '~utils/tokens';
 import { setQueryParamOnUrl } from '~utils/urls';
-import { ACTION_TYPE_FIELD_NAME } from '~v5/common/ActionSidebar/consts';
 import ColonyDashboardHeader from '~v5/common/ColonyDashboardHeader';
-import DonutChart from '~v5/common/DonutChart';
-import TeamReputationSummaryRow from '~v5/common/TeamReputationSummary/partials/TeamReputationSummaryRow';
-import EmptyWidgetState from '~v5/common/WidgetBox/partials/EmptyWidgetState';
-import WidgetBoxList from '~v5/common/WidgetBoxList';
 import Link from '~v5/shared/Link';
-import MessageNumber from '~v5/shared/MessageNumber';
 import Modal from '~v5/shared/Modal';
-import ProgressBar from '~v5/shared/ProgressBar';
-import TitleWithNumber from '~v5/shared/TitleWithNumber';
-import UserAvatars from '~v5/shared/UserAvatars';
 
-import { useDashboardHeader, useGetHomeWidget } from './hooks';
+import { useDashboardHeader } from './hooks/useDashboardHeader';
+import Agreements from './partials/Agreements';
+import Members from './partials/Members';
+import Objective from './partials/Objective';
+import ReputationChart from './partials/ReputationChart';
+import TokenBalance from './partials/TokenBalance';
+import TotalActions from './partials/TotalActions';
```

Comment:
Could we add a `index` file that exports all of these? That way we can reduce all of these imports to just one that imports multiple components